test.js

```diff
@@ -440,6 +440,55 @@ describe('cli', function () {
     })
   })
 
+  describe('zero-mode', function () {
+    it('works with patch release', function () {
+      writePackageJson('0.1.0')
+      commit('feat: first commit')
+      shell.exec('git tag -a v0.1.0 -m "my awesome first release"')
+      commit('fix: patch release')
+
+      execCli('--zero-mode').code.should.equal(0)
+
+      JSON.parse(fs.readFileSync('package.json', 'utf-8')).version.should.equal('0.1.1')
+    })
+
+    it('works with feature release', function () {
+      writePackageJson('0.1.0')
+      commit('feat: first commit')
+      shell.exec('git tag -a v0.1.0 -m "my awesome first release"')
+      commit('feat: feature release')
+
+      execCli('--zero-mode').code.should.equal(0)
+
+      JSON.parse(fs.readFileSync('package.json', 'utf-8')).version.should.equal('0.1.1')
+    })
+
+    if (process.platform !== 'win32') {
```

Comment:
I think the 

```js
if (isWindows) {
  return this.skip();
}
```

pattern we use in istanbuljs is a bit cleaner. I think it's worth the extra devDep

Reply:
Done